src/main/webapp/movement.js

```diff
@@ -0,0 +1,40 @@
+/** The function that calls the two functions below */
+function callsotherfunctions(){
+    function loadgetmovement();
+    function creategetmovementElement(movement);
+}
+
+
+/** Fetches movement hashtags from the server and adds them to the DOM. */
+function loadgetmovement() {
+  fetch('/get-movement').then(response => response.json()).then((movements) => {
+    const movementListElement = document.getElementById('movement-list');
+    movements.forEach((movement) => {
+      movementListElement.appendChild(createMovementElement(movement));
+    })
+  });
+}
+
+
+function creategetmovementElement(movement) {
```

Comment:
i think you want to take our template from StopAsianHate and fill out those fields instead of starting from scratch.

Reply:
I changed it to the following, but I do not know if it is correct: 
function getmovementElement(movement) {
    getmovementElement.name = 'name';
    getmovementElement.description = 'description';
    getmovementElement.hashtag = 'hashtag';
    getmovementElement.video = 'video';
    getmovementElement.image = 'image';
    getmovementElement.donatation = 'donatation';
    getmovementElement.resources = 'resources'; 
}

Reply:
In addition, I also added the twitter bot function that was previously on there, but I do not know what to do with the for loop: 
for (var i = 0, keys = Object.keys(LinkedHashMap), ii = keys.length; i < ii; i++) {
  console.log('key : ' + keys[i] + ' val : ' + LinkedHashMap[keys[i]]);
}

async function getBot() {
    const responseFromServer = await fetch('/twitter-bot' + new URLSearchParams({
    hashtag: '#stopasianhate',

  }));
  const textFromResponse = await responseFromServer.text();

  const botContainer = document.getElementById('bot');
  botContainer.innerText = textFromResponse;
}